app/services/service_provider_seeder.rb

```diff
@@ -42,8 +42,7 @@ def run
   def service_providers
     file = Rails.root.join('config', 'service_providers.yml').read
     file.gsub!('%{env}', deploy_env) if deploy_env
-    content = ERB.new(file).result
-    YAML.safe_load(content).fetch(rails_env)
+    YAML.safe_load(file).fetch(rails_env)
```

Comment:
**question:** Is there a reason we still need `.fetch(rails_env)` at this point? Do we need env nesting in our config files? Will they ever not be `prod`?

Reply:
@julialeague it will for local development and testing!

Reply:
Ah yes, I forgot about the seeder spec!